test/core/clock/chain_epoch_clock_test.cpp

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "clock/impl/chain_epoch_clock_impl.hpp"
+
+#include <gtest/gtest.h>
+#include "testutil/outcome.hpp"
+
+using namespace fc::clock;
```

Comment:
It triggers warning 
"do not use namespace using-directives; use using-declarations instead [google-build-using-namespace,-warnings-as-errors]"
Isn't it?

Reply:
@kamilsa, is it ok to use `using namespace` in tests ([1](https://github.com/filecoin-project/cpp-filecoin/blob/master/test/core/common/blob_test.cpp#L10), [2](https://github.com/filecoin-project/cpp-filecoin/blob/master/test/core/common/hexutil_test.cpp#L11), [3](https://github.com/filecoin-project/cpp-filecoin/blob/master/test/core/storage/leveldb/leveldb_integration_test.cpp#L17), [4](https://github.com/filecoin-project/cpp-filecoin/blob/master/test/core/storage/leveldb/leveldb_fs_test.cpp#L14))?
Is it ok that tests are [ignored by clang-tidy](https://github.com/filecoin-project/cpp-filecoin/blob/master/.clang-tidy-ignore#L4)?

Reply:
I prefer to use using declarations as Alexey suggested. This helps to avoid possible conflicts in names. If it is not a big deal please fix it